### PR TITLE
Adds Option to see only Active Courses

### LINKED ISSUE #418: Show only Active Courses
Since the number of courses is increasing, we need a way to show only Active Courses.

### PR DESCRIPTION
**Pull request by Akshen**
- Changes in courses.html
- Adds option to see only Active Courses in both My Courses and Allotted Courses
- Tooltip over Show Active Courses
- Highlights Selected Option in SideBar
- Closes #418

### CODE REVIEW

File: yaksh/templates/yaksh/courses.html [hunk 1]
```diff
@@ -3,23 +3,69 @@
...
     $(document).ready(function(){
+    $('li').click(function(){
+        $('li').removeClass("active");
+        $(this).addClass("active");
+        });
+    });
+
+    //Show Info on hover over Show Active Courses
+    $(document).ready(function(){
+     $('[data-toggle="tooltip"]').tooltip();
```
Comment (maheshgudi) [inline yaksh/templates/yaksh/courses.html]: I think the link name should be self-explanatory, this is completely unnecessary

**Comment by prathamesh920** [comment]
This LGTM.
But the template is growing indefinitely and also there is lot of redundancy.
So I have created issue #420  for the same.

File: yaksh/templates/yaksh/courses.html [hunk 2]
```diff
@@ -3,23 +3,69 @@
...
         });
         $("#link_allotted_courses").click(function() {
+            flag = 1;
             if ($("#created_courses").is(":visible")){
                 $("#created_courses").toggle();
             }
             if (!$("#allotted_courses").is(":visible")){
                 $("#allotted_courses").toggle();
             }
+            if ($("#show_only_active").is(":visible")){
```
Comment (maheshgudi) [inline yaksh/templates/yaksh/courses.html]: Why are you doing this? Bootstrap does this by default.

File: yaksh/templates/yaksh/courses.html [hunk 3]
```diff
@@ -46,6 +92,10 @@
             <li>
                 <a href="{{URL_ROOT}}/exam/manage/courses/all_learning_module">
                 Add/View Modules</a>
+            </li><li>
+                <a href="#" id="link_show_only_active" data-toggle="tooltip" title="Show Active courses of current window">
```
Comment (maheshgudi) [inline yaksh/templates/yaksh/courses.html]: Tooltip is barely any different than the link name.

File: yaksh/templates/yaksh/courses.html [hunk 4]
```diff
@@ -180,14 +230,146 @@
...
+{% if type == "courses" %}
+<div id="show_only_active" style="display: none;">
+{% if not courses %}
+    <center><h4> No new Courses created </h4></center>
+{% else %}
+<div class="col-md-offset-2 main">
+    <center><h3> Course(s) Created</h3></center>
+    <table id="show_only_active" class="table table-bordered">
+    <tr>
+    <th>Courses</th>
```
Comment (maheshgudi) [inline yaksh/templates/yaksh/courses.html]: If there are courses but are not active, then we see this table with module and courses as th tags. Fix this

File: yaksh/templates/yaksh/courses.html [hunk 5]
```diff
@@ -313,12 +495,151 @@ <h4>{{course.creator.get_full_name.title}}</h4>
...
     </tr>
     </table>
-</div>
+    </div>
 {% endif %}
 </div>
 {% endif %}
 <!-- End if all Courses  -->
 
+<!-- ***********************Show Only Active Allotted Courses******************* -->
```
Comment (maheshgudi) [inline yaksh/templates/yaksh/courses.html]: The same table is shown for active allotted courses and my active. This is redundant

**Comment by Akshen** [comment]
Backend to be restructured

**Pull request closed without merging**